packages/runtime/src/lib/constants.ts

```diff
@@ -536,36 +536,121 @@ export enum vrfVerifyFieldTypes {
 }
 
 export enum TxFieldEnum {
+	Amount = "Amount",
+	Accounts = "Accounts",
+	AcctBalance = "AcctBalance",
+	AcctMinBalance = "AcctMinBalance",
```

Comment:
This should only have txFields. Some of them are just opcode parameters which are not part of a tx. Lets move it to a separate enums

Reply:
Done